docs/data-model.asciidoc

```diff
@@ -175,6 +175,125 @@ For a full list of tracked metrics, see the relevant agent documentation:
 * {apm-py-ref-v}/metrics.html[Python]
 * {apm-ruby-ref-v}/metrics.html[Ruby]
 
```

Comment:
The overview content above this line might also need some work to better differentiate between the APM-defined metrics and runtime/infra metrics.

Reply:
Yes! I was going to ask for some clarification. Should the two types of metrics be organized under two subheadings like...

>* Metrics
>    * Runtime metrics
>    * Additional metrics / Calculated metrics  / APM-defined metrics 

... or are "Runtime metrics" synonymous with "Metrics" as its used in the page title  (and "Additional metrics / APM-defined metrics" are a subtype of "Metrics")? Based on the legacy docs, it seems like the latter so that's how I treated it in [`2afc1c0`](https://github.com/elastic/apm-server/pull/7299/commits/2afc1c06a7938809247e06e202630337e8168a55).

Reply:
@marclop would you mind commenting on this question too?

Reply:
That sounds good to me. From what I can see in our current docs, it looks like the current metrics section mostly talks about APM Agent runtime (Go, Java) and system metrics. Both metrics are a Subtype of "Metrics", just some of them are generated in different places. APM Agents will send some metrics and APM Server will do minimal processing on those, but also the APM Server creates metrics from the Transactions and Spans it receives to power the APM UI more efficiently.

The current change looks fine, but `APM-defined metrics` is a little general, perhaps? `Calculated Metrics` maybe a little more precise.

Reply:
> The overview content above this line might also need some work to better differentiate between the APM-defined metrics and runtime/infra metrics.

@bmorelli25 is something like this what you had in mind? 

>## Metrics
>
>**Metrics** measure the state of a system by gathering information on a regular interval. There are two types of APM metrics:
>
>* **System metrics**: APM agents pass along information that reflects the status of the system with minimal processing.
>* **Calculated metrics**: APM Server takes information provided by APM agents and processes it into metrics that power visualizations in the APM App.
>
>### System metrics
>
>[existing content on [Data Model > Metrics](https://www.elastic.co/guide/en/apm/guide/current/data-model-metrics.html)]
>
>### Calculated metrics
>
>[content copied from legacy docs]
>

Some notes:
 
* I structured the first sentence to mirror the other Data Model guides (for example [Spans](https://www.elastic.co/guide/en/apm/guide/current/data-model-spans.html) and [Transactions](https://www.elastic.co/guide/en/apm/guide/current/data-model-transactions.html)), which start with a definition of the information type. I'm not sure the definition I came up with is great, though. 😬 
*  I called the first metric type "System metrics", but I'm not sure that's the right term. Throughout the conversation in this PR we've used "system", "infrastructure", "application", "runtime" to describe this first bucket of metrics. Is there a more general term that would capture all these?
* I attempted a general description of these two kinds of metrics that could definitely be refined by someone with more knowledge! The intent is to give the reader an idea of what to expect, but not get into detail until the sections below.

Reply:
> I structured the first sentence to mirror the other Data Model guides...

Looks great.

> I called the first metric type "System metrics"...

I'm not sure what the best word is here either. Let's run with what you have for now. "System" seems generic enough, but the dev team might have a better suggestion when it goes to review.

> I attempted a general description of these two kinds of metrics

A few suggestions:

> APM agents pass along information that reflects the status of the system with minimal processing.

APM agents do "pass along" this information, but I think it's more important that they collect it. I think something simple like this might work:
"APM agents collect basic infrastructure and application metrics."

> Calculated metrics: APM Server takes information provided by APM agents and processes it into metrics that power visualizations in the APM App.

This one is harder to give an idea of what to expect without too many details. A couple of thoughts:
* It's not just the APM Server that creates these metrics. AFAIK, Breakdown metrics are generated by APM agents.
* I think it's important to note that these metrics are calculated from trace events

We could go super simple with something like this:
"Aggregated trace event metrics used to power visualizations in the APM app."